src/bootstrap/src/core/build_steps/compile.rs

```diff
@@ -2020,7 +2020,9 @@ impl Step for Assemble {
                 builder.info(&format!("Creating a sysroot for stage{stage} compiler (use `rustup toolchain link 'name' build/host/stage{stage}`)", stage=target_compiler.stage));
             }
 
-            maybe_install_llvm_bitcode_linker();
+            let mut precompiled_compiler = target_compiler;
+            precompiled_compiler.forced_compiler(true);
+            maybe_install_llvm_bitcode_linker(precompiled_compiler);
```

Comment:
Note that the comment for `Compiler::forced_compiler` says

> Indicates whether `compiler_for` was used to force a specific compiler stage.

In this case, it's "forced" in that it's because it's a CI rustc, not because it's a resolved `rustc` produced from `compiler_for`, right?

Reply:
That doc should be updated.